### PR TITLE
Added Ownership of mongod.conf To MongoDB User

### PR DESCRIPTION
**Pull request by ddaskal**
Also fixed ceilometer service type in endpoints role.

### CODE REVIEW

File: roles/endpoints/defaults/main.yml [hunk 1]
```diff
@@ -105,50 +105,50 @@ endpoints:
...
-      backend_api: 6384                                                                 
+    url:
+      admin: https://{{ fqdn }}:6385
+      internal: https://{{ fqdn }}:6385
+      public: https://{{ fqdn }}:6385
+    port:
+      backend_api: 6384
       haproxy_api: 6385
   swift:
     name: swift
```
Comment (j2sol) [inline roles/endpoints/defaults/main.yml]: all the changes in this file seem unrelated. Please make this a different commit/pr

**Comment by ddaskal** [comment]
Atom automatically removed all blank spaces after the last character for ever line when I saved the file. I only made one change per file:
1. Added 'owner=mongodb' for the mongod.conf file
2. Line 133, changed ceilometer 'type=metering' instead of 'type=Telemetry', to match the keystone service.

**Comment by j2sol** [comment]
retest

**Comment by ddaskal** [comment]
retest
